autorest/codegen/templates/README.md.jinja2

```diff
@@ -0,0 +1,110 @@
+{% if package_mode == "mgmtplane" -%}
+# Microsoft Azure SDK for Python
+
+This is the Microsoft {{package_pprint_name}} Client Library.
+This package has been tested with Python 3.6+.
+For a more complete view of Azure libraries, see the [azure sdk python release](https://aka.ms/azsdk/python/all).
+
+# Usage
+
+To learn how to use this package, see the [quickstart guide](https://aka.ms/azsdk/python/mgmt)
+
+For docs and references, see [Python SDK References](https://docs.microsoft.com/python/api/overview/azure)
+Code samples for this package can be found at [{{package_pprint_name}}](https://docs.microsoft.com/samples/browse/?languages=python&term=Getting%20started%20-%20Managing&terms=Getting%20started%20-%20Managing) on docs.microsoft.com.
+Additional code samples for different Azure services are available at [Samples Repo](https://aka.ms/azsdk/python/mgmt/samples)
+
+# Provide Feedback
+
+If you encounter any bugs or have suggestions, please file an issue in the
+[Issues](https://github.com/Azure/azure-sdk-for-python/issues)
+section of the project. 
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2F{{package_name}}%2FREADME.png)
+{% else %}
+# {{ package_pprint_name }} client library for Python
+<!-- write necessary description of service -->
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please 
+refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+
+## Getting started
+
+### Installating the package
+
+```bash
+python -m pip install {{ package_name }}
```

Comment:
That's debatable, we saw people confused because they had several Python installed, and `python` and `pip` were not referencing the same python interpreter. I would suggest to talk about venv instead, since this is a more efficient to ensure consistency between python and pip